### ISSUE TITLE
Funds stolen

### ISSUE THREAD

**Issue by system123** [norbertvdberg/iotaseed#8]
Numerous reports are going out, and I am affect too from using your website to generate a secure seed for IOTA wallet. This is either you which is involved, or there is a severe security flaw in your system. Just bringing this to your attention.

**Comment by peschu123** [comment]
I can personally approve this shit...
There can always be another reason, but I'm 95% sure that this site steals seeds ... Maybe it is not the same like the source code on github ---> iotaseed.io is malicious in my opinion ....

**Comment by system123** [comment]
Perhaps, but the github repo promotes it. Therefore the author of the repo is in on it.

**Comment by hardcqre** [comment]
I can confirm, my funds have been stolen in two attempts. One transaction went through this morning (GMT+1)

**Comment by system123** [comment]
Yup, site send the generated seed back to the server via a get request under the guise of an email address.

**Comment by hardcqre** [comment]
The Code ist not There anymore though, is it?

**Comment by hardcqre** [comment]
Seems to be coordinated over several pages. Funds End up here
https://thetangle.org/transaction/UO9BUTRUVJDTKZDZQCOXJISTVNEI9BIWBHPHVMIWFGJFUYXVBIIUDNJYNHPTUMSJVHCZNMXOWTBY99999

**Comment by system123** [comment]
https://twitter.com/eukaryote314/status/953839632206020608

**Comment by hardcqre** [comment]
Yes but its a different page *.co and *.io

**Comment by system123** [comment]
True but I was affected and used .io, so they are likely linked.

**Comment by hardcqre** [comment]
Its iotaseed.net which Looks exactly The Same as iotaseed.io
So iotaseed.io most likely isnt scammy.

**Comment by system123** [comment]
I double checked by going through my browser history, I have never visited iotaseed.net or anything other than iotaseed.io so not sure why my funds were stolen then.

**Comment by HeyItsMerlin** [comment]
My funds have been stolen too this night. I used iotaseed.io to generate the seed last month. I don’t know any other reason how they should be stolen.

**Comment by system123** [comment]
Exactly the same situation. Generated my seed on December 27.

**Comment by system123** [comment]
If the transaction is pending you can possibly save it. 
- Log into your iota wallet and send the funds to another address which is secure (create a new safe seed). Then go to the IOTA discord discussion and ask people there to do the POW urgently to save your funds.

**Comment by ceej64** [comment]
Thanks, but it doesnt let me... It says "Error: Private Key reuse Detected!"

You are attempting to sign a transaction with input that have an already been used. IOTA uses Winternitz one-time signature (W-OTS) scheme, due to it's `one-time` nature, the security of funds in an address decreases rapidly if you re-sign new transactions using the same key. Please wait for previous transaction to confirm, e.g. by promoting/reattaching, before sending another transaction.

**Comment by system123** [comment]
Visit the discord chat, some people there might be able to help you
further. There are people who have saved their funds

**Comment by ceej64** [comment]
Good shout, some very nice ppl over there helped me get my funds back.

**Comment by system123** [comment]
You were one of the lucky ones whose transaction hadn't confirmed yet. Glad
you got your funds back.

**Comment by Bluephantom3** [comment]
My funds were stolen last night just before midnight Australian EST. 

I saved iotaseed.io to my desktop and disconnected the internet. Reopened it and generated the seed. I used a printscreen to create a paper wallet completely offline. After printing I deleted everything on the PC.

I manually typed the seed into the IOTA lite wallet to send funds to it.

The seed generator must not be random.

**Comment by system123** [comment]
Seems it used pre-generated seeds.

**Comment by redconfetti** [comment]
I generated my seed on Dec 22nd. I surely used iotaseed.io. No other site is in my history. Funds were stolen before midnight last night also (19/01/2018 23:12). I also had checked the network tab to ensure that it wasn't transmitting the data via the Network tab in my browser.

https://thetangle.org/transaction/JGOCWWP9GKLLPYMRDCYZLXRXLTWJXVZWWDYLIIDWDSEZSDDIRKRDQAHTEOS9YFDFKFPJNNAFAHLE99999

**Comment by rllybadcryptotrdr** [comment]
I generated a seed back in August. I'm pretty sure I used the offline version, as "online isn't secure," lol.  So, 5.5 months later, I get robbed. This was a scam. THIS. They were all related. Everyone is getting robbed at the same time. Right now.
